Added Repeato step doc (#412)
appcircleio/appcircle-docusaurus: docs/workflows/common-workflow-steps/repeato-test-runner.md

---
title: Repeato Test Runner
description: Repeato Test Runner facilitates the execution of automated tests for mobile applications directly within Appcircle.
tags: [repeato, test-runner, mobile, automation]
sidebar_position: 11
---

import Screenshot from '@site/src/components/Screenshot';

# Repeato Test Runner

[Repeato](https://www.repeato.app) is a test automation platform designed for mobile applications. It enables developers to create, manage, and execute automated tests for mobile apps across different platforms and devices. Repeato supports various testing frameworks and provides features for test script creation, test execution, result analysis, and reporting. It helps streamline the testing process, improve test coverage, and enhance the overall quality of mobile applications.

> The **Repeato Test Runner** is a service integrated into the CI/CD workflow of Appcircle. It facilitates the execution of automated tests for mobile applications directly within Appcircle. This service allows developers to validate the functionality and performance of their mobile applications before deploying them, ensuring high-quality releases.

Reviewer: ```suggestion
The **Repeato Test Runner** integrates as a service within the Appcircle CI/CD workflow, streamlining the execution of automated tests directly within Appcircle. This service enables developers to validate the functionality and performance of their mobile applications before deployment, ensuring releases of high quality.
```

For more information, please check out this blog post:

https://appcircle.io/blog/streamline-project-integration-and-test-automation-with-repeato-and-appcircle

### Prerequisites

Before executing the **Repeato Test Runner** workflow step, certain prerequisite workflow steps must be completed:

| Prerequisite Workflow Step | Description                                                                                   |
| -------------------------- | --------------------------------------------------------------------------------------------- |
| [**Git Clone**](/workflows/common-workflow-steps/#git-clone) | The repository that needs to be built must be fetched from the branch. Upon completion of the **Git Clone** step, it generates the `AC_REPOSITORY_DIR` variable, which is then used as the input for the Android Build step. |

> 

Reviewer 2: And you should provide other platform steps, if it is need.

> <Screenshot url='https://cdn.appcircle.io/docs/assets/common-workflow-components-repeato_1.png'/>

Reviewer 2: Would it be better if step has place right after git clone?
